src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,114 @@
+package frc.trigon.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.ctre.phoenix6.signals.SensorDirectionValue;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import static frc.trigon.robot.utilities.Conversions.offsetRead;
```

Comment:
Never ever ever ever use a static import. Write `Conversions.offsetRead()` instead.

Reply:
This was not fixed.

Reply:
yes it was